.gitattributes

```diff
@@ -2,6 +2,8 @@
 /.github export-ignore
 /.gitignore export-ignore
 /.travis.yml export-ignore
+/CODE_OF_CONDUCT.md
```

Comment:
Shouldn't this have `export-ignore` after it? @jdpedrie

Reply:
@bshaffer ups yes, I forgot that.